Pod/Classes/PINRemoteImageManager.m

```diff
@@ -751,6 +751,25 @@ - (void)downloadImageWithURL:(NSURL *)url
     [self unlock];
 }
 
+-(BOOL) insertImageDataIntoCache:(nonnull NSData*)data withURL:(nonnull NSURL *)url processorKey:(nullable NSString *)processorKey {
+  
+  if (url != nil) {
+    NSString *key = [self cacheKeyForURL:url processorKey:processorKey];
+    
+    PINRemoteImageManagerDownloadOptions options = PINRemoteImageManagerDownloadOptionsSkipDecode & PINRemoteImageManagerDownloadOptionsSkipEarlyCheck;
+    PINRemoteImageMemoryContainer *container = [[PINRemoteImageMemoryContainer alloc] init];
+    container.data = data;
+    
+    return [self materializeAndCacheObject: container cacheInDisk: data additionalCost:0 key:key options:options outImage: nil outAltRep: nil];
```

Comment:
Just to clarify on what cost is here: this is a number which can be set arbitrarily high which would increase the likelihood that an object gets evicted from the cache?